example/hyprland.conf

```diff
@@ -76,6 +76,11 @@ general {
     allow_tearing = false
 
     layout = dwindle
+    #SmartGaps( automatic hides gaps if only one window is on the screen )
+    ##workspace = w[t1], gapsout:0, gapsin:0, border: 0, rounding:0
```

Comment:
`##` means literal `#`. Should be `# <content>` everywhere instead. Mind the space for legibility.